pystiche_papers/li_wand_2016/_data.py

```diff
@@ -70,6 +70,20 @@ def extra_repr(self) -> str:
         return f"size={self.size}"
 
 
+def license_info(original: Optional[str] = None) -> str:
+    license_text = (
+        "The image is part of a repository that is published under the MIT License "
+        "(MIT) "
+        "(https://github.com/chuanli11/CNNMRF/blob/fddcf4d01e2a6ce201059d8bc38597f74a09ba3f/License#L1)."
+    )
+    if original:
+        license_text = (
+            f"{license_text} The original was probably downloaded from {original}. "
+            f"Proceed at your own risk."
```

Comment:
Should we have this disclaimer also for Gatys 2017?

Reply:
We have integrated this disclaimer for Gatys 2017, but you probably mean ulyanov and johnson. But this is only added if original is given. I would say that it would make sense to integrate this everywhere at the end of the licence by default.

Reply:
```python
def license(original: Optional[str] = None) -> str:
    license_text = (
        "The image is part of a repository that is published under the MIT License "
        "(MIT) "
        "(https://github.com/chuanli11/CNNMRF/blob/fddcf4d01e2a6ce201059d8bc38597f74a09ba3f/License#L1)."
    )
    if original:
        license_text = (
            f"{license_text} The original was probably downloaded from {original}. "
        )
    return f"{license_text} Proceed at your own risk."
```